securetea/core.py

```diff
@@ -80,26 +106,55 @@ def __init__(self):
             cred['debug']
         )
 
-        self.twitter = secureTeaTwitter.SecureTeaTwitter(
-            cred['twitter'],
-            cred['debug']
-        )
+        if cred_provided is False:
+            self.logger.log(
+                "None of the parameters configured. Exiting...",
+                logtype="error"
+            )
+            sys.exit(0)
+            
+        if not self.twitter.enabled:
+            self.twitter = secureTeaTwitter.SecureTeaTwitter(
+                cred['twitter'],
+                cred['debug']
+            )
 
         if self.twilio_provided:
             self.twilio = secureTeaTwilio.SecureTeaTwilio(
                 cred['twilio'],
                 cred['debug']
             )
 
-        if not self.twitter.enabled:
-            self.logger.log(
-                "Twitter not configured properly. Exiting...",
-                logtype="error"
+        self.logger.log("Welcome to SecureTea..!! Initializing System")
+
+        if self.twitter_provided:
+            self.twitter = secureTeaTwitter.SecureTeaTwitter(
+                cred['twitter'],
+                cred['debug']
             )
-            sys.exit(0)
-        else:
-            self.logger.log("Welcome to SecureTea..!! Initializing System")
-            self.twitter.notify("Welcome to SecureTea..!! Initializing System")
+
+            if not self.twitter.enabled:
+                self.logger.log(
+                    "Twitter not configured properly. Exiting...",
```

Comment:
Are we exiting here? Only exit if none of the notification configured.
So, The message should be "None of the notifications configured. Exiting. "

Reply:
No, we aren't exiting here. Just a message that Twitter notifications not enabled. I'll update the log message.